routes/web.php

```diff
@@ -15,67 +15,71 @@
 
 // Task 1: point the main "/" URL to the HomeController method "index"
 // Put one code line here below
-
+Route::get('/',[HomeController::class,'index']);
 
 // Task 2: point the GET URL "/user/[name]" to the UserController method "show"
 // It doesn't use Route Model Binding, it expects $name as a parameter
 // Put one code line here below
-
+ 
+Route::get('/user/{name}',[UserController::class,'show']);
 
 // Task 3: point the GET URL "/about" to the view
 // resources/views/pages/about.blade.php - without any controller
 // Also, assign the route name "about"
 // Put one code line here below
-
+Route::get('/about',function(){
+return view('about')
```

Comment:
The [return](https://www.php.net/manual/en/function.return.php) [statement](https://www.php.net/manual/en/control-structures.intro.php) is missing its ending semicolon.